Foodfeedback/Controllers/LoginController.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Foodfeedback.DTO;
+using Foodfeedback.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Foodfeedback.Controllers
+{
+    [Route("api/login")]
+    public class LoginController
+    {
+        private readonly ILoginService _loginService;
+        public LoginController(ILoginService loginService)
+        {
+            _loginService = loginService;
+        }
+        [HttpPost]
+        public bool GetUserdetails([FromBody]UserDTO userDTO)
+        {
```

Comment:
Include Exception handling and on any exception return an appropriate status code